src/Services/Jira/JiraProject.php

```diff
@@ -53,7 +59,7 @@ public static function getIssues(string $projectKey, int $startAt = 0, int $maxR
         return $issueSearchQueryResult;
     }
 
-    private static function mapIssues(mixed $data): Issue
+    private function mapIssues(mixed $data): Issue
```

Comment:
I'm good with this changing to no longer be a static function, but can you please update the static call on L56 to be `$this->mapIssues($issue)`?

Reply:
So this was a brain fart on my part. LGTM!